cmd/notification.go

```diff
@@ -346,11 +344,11 @@ func (sys *NotificationSys) ServerInfo(ctx context.Context) []ServerInfo {
 // GetLocks - makes GetLocks RPC call on all peers.
 func (sys *NotificationSys) GetLocks(ctx context.Context) []*PeerLocks {
 	var idx = 0
-	locksResp := make([]*PeerLocks, len(sys.peerRPCClientMap))
+	locksResp := make([]*PeerLocks, len(sys.peerHosts))
 	var wg sync.WaitGroup
-	for addr, client := range sys.peerRPCClientMap {
+	for _, addr := range sys.peerHosts {
```

Comment:
We don't need idx anymore it was used with map previously, since now we have peerHosts as slice we can simply do 

```
for idx, addr := range sys.peerHosts {
```